app/src/main/java/ch/epfl/sweng/swenggolf/database/FakeDatabase.java

```diff
@@ -185,6 +186,29 @@ public void readOffers(@NonNull ValueListener<List<Offer>> listener,
                 listener, categories);
     }
 
+    @Override
+    public void readFollowers(@NonNull ValueListener<Map<String, List<String>>> listener) {
+        Map<String, List<String>> userFollowing = new HashMap<>();
+        for (Map.Entry<String, Object> entry : database.entrySet()) {
+            if (entry.getKey().startsWith(FOLLOWERS_PATH)) {
```

Comment:
Do you mean before or after? I did remember the bug and took care to remove the '/' that is at the beginning of it, because FOLLOWERS_PATH corresponds to "/followers". Or maybe that I didn't understand the previous issue there was with offers?

Reply:
if some entry is something like /followersSub then it begins by /followers whereas it doesn't start with /followers/, so just put FOLLOWERS_PATH + "/". Correct me if you took care of this case somewhere else.